### ISSUE TITLE
getting an error after connecting

### ISSUE THREAD

**Issue by sarunkums** [ruby-openstack/ruby-openstack#82]
i'm attempting to perform actions after connecting.  Authorization returns true, but after that, I get the following error:

`C:/Ruby193/lib/ruby/gems/1.9.1/gems/openstack-2.0.2/lib/openstack/connection.rb:228:in `headerprep': uninitialized constant OpenStack::VERSION (NameError)
        from C:/Ruby193/lib/ruby/gems/1.9.1/gems/openstack-2.0.2/lib/openstack/connection.rb:172:in `csreq'
        from C:/Ruby193/lib/ruby/gems/1.9.1/gems/openstack-2.0.2/lib/openstack/connection.rb:216:in `req'
        from C:/Ruby193/lib/ruby/gems/1.9.1/gems/openstack-2.0.2/lib/openstack/compute/connection.rb:278:in `api_extensions'
        from openstackPoC.rb:13:in `<main>'`

Any idea what the problem is here?  in auth_url, I've specified "https://my.url:5000/v2.0/

**Comment by parthnaik** [comment]
Same problem here, any solution to this?

**Comment by pcandme** [comment]
Same here. No problem on connection. Try #servers and have NameError:

```
NameError: uninitialized constant OpenStack::VERSION
/Users/pcandme/.rvm/gems/ruby-2.3.0/gems/openstack-2.0.2/lib/openstack/connection.rb:228:in `headerprep'
/Users/pcandme/.rvm/gems/ruby-2.3.0/gems/openstack-2.0.2/lib/openstack/connection.rb:172:in `csreq'
/Users/pcandme/.rvm/gems/ruby-2.3.0/gems/openstack-2.0.2/lib/openstack/compute/connection.rb:49:in `list_servers'
```

Mac OS X, Ruby 2.3.0.

**Comment by pcandme** [comment]
I just added a file, 'version.rb' in lib/openstack/, whose content is:

```
module OpenStack
  VERSION = "2.0.2"
end
```

and added a require statement in ib/openstack.rb as follows:

```
...
  $:.unshift(File.dirname(__FILE__))
  require 'openstack/connection'
  require 'openstack/version' ## <== Added!!
  require 'openstack/compute/connection'
...
```

This solved my problem.

**Comment by BirknerAlex** [comment]
Hello,

sorry for the inconvenience. The issue has been solved in the latest version 2.2.0 which has been already released on rubygems.org.

Regards,
Alexander
